roles/monitoring-common/defaults/main.yml

```diff
@@ -11,4 +11,4 @@ monitoring:
     vhost: /sensu
     username: sensu
     password: sensu
-  cluster_prefix: "ip a | grep :ucarp > /dev/null &&"
\ No newline at end of file
+  cluster_prefix: "ip a | grep :ucarp > /dev/null"
```

Comment:
There is still something wrong with this. The && is supposed to be there according to @paulczar